### PR TITLE
Transfer static content of subscriptions to new Play app

### PR DESCRIPTION
**Pull request by tudorraul**

### CODE REVIEW

**Comment by rtyley** [comment]
> packages links on the front page are pointing to app's pages instead of guardiansubscriptions.co.uk

Unfortunately, I think they need to be linking to the legacy systems for the time being... we want to be able to switch over subscribe.theguardian.com to point to our new backend as soon as possible (ie days), so we can't direct users to our new pages until they're ready!

**Comment by tudorraul** [comment]
I see, ok, I'll switch them back.

File: app/views/index.scala.html [hunk 1]
```diff
@@ -1,9 +1,126 @@
...
+                </div>
+                <div class="block__footer">
+                    <a class="button button--large button--primary" href="http://www.theguardian.com/mobile/2014/may/29/the-guardian-for-mobile-and-tablet" data-test-id="subscriptions-uk-app-store">
+                        Find out more</a>
+                </div>
+            </div>
+        </div>
+    </div>
 
+    <script type="text/javascript" src="https://go.affec.tv/j/54e4b48a0189603f6b557597"></script>
```
Comment (marchibbins) [inline app/views/index.scala.html]: This should be an analytics fragment.

File: app/views/index.scala.html [hunk 2]
```diff
@@ -1,9 +1,126 @@
...
+        </div>
+    </div>
+    <div class="gs-container">
+        <div class="row row--padding-top row--items-3">
+            <div class="row__item block">
+                <h2 class="block__title">Guardian Weekly</h2>
+                <div class="block__info">
+                    <ul class="block__list">
+                        <li class="block__list-item">
+                            A selection of international editorial from the Guardian, Observer, Washington Post and Le Monde</li>
```
Comment (marchibbins) [inline app/views/index.scala.html]: Don't start this on a new line, renders unnecessary whitespace in browser.

File: app/views/index.scala.html [hunk 3]
```diff
@@ -1,9 +1,126 @@
...
+                        <li class="block__list-item">
+                            A unique blend of international news, politics, culture and comment</li>
```
Comment (marchibbins) [inline app/views/index.scala.html]: As above.

File: app/views/shipping/index.scala.html [hunk 4]
```diff
@@ -0,0 +1,213 @@
+@(shipping: model.ShippingDetails)
+
+@main(s"${shipping.name} | Choose a Package | Subscriptions | The Guardian", footer = fragments.footer.uk_faq()) {
```
Comment (marchibbins) [inline app/views/shipping/index.scala.html]: This should use `fragments.footer.intl_faq()`

File: app/controllers/Shipping.scala [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+      everyday = Some("https://www.guardiandirectsubs.co.uk/Delivery/details.aspx?package=EVERYDAY%2B"),
+      sixday = Some("https://www.guardiandirectsubs.co.uk/Delivery/details.aspx?package=SIXDAY%2B"),
+      weekend = Some("https://www.guardiandirectsubs.co.uk/Delivery/details.aspx?package=WEEKEND%2B"),
+      sunday = Some("https://www.guardiandirectsubs.co.uk/Delivery/details.aspx?package=SUNDAY%2B")
+    ))
+  }
+
+  val deliveryPaper = Action { request =>
+    index(request, ShippingDetails(
+      name = "Paper home delivery subscription",
```
Comment (marchibbins) [inline app/controllers/Shipping.scala]: Capital H on "Home"

File: app/controllers/Shipping.scala [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+      packageType = "paper",
+      everyday = Some("https://www.guardiansubscriptions.co.uk/Voucher?prom=faa03&pkgcode=ukx00&title=gv7&skip=1"),
+      sixday = Some("https://www.guardiansubscriptions.co.uk/Voucher?prom=faa03&pkgcode=ukx00&title=gv6&skip=1"),
+      weekend = Some("https://www.guardiansubscriptions.co.uk/Voucher?prom=faa03&pkgcode=ukx00&title=gv2&skip=1")
+    ))
+  }
+
+  val deliveryPaperDigital = Action { request =>
+    index(request, ShippingDetails(
+      name = "Paper + digital home delivery subscription",
```
Comment (marchibbins) [inline app/controllers/Shipping.scala]: "Paper + Digital Home Delivery Subscription"

File: app/controllers/Shipping.scala [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+      everyday = Some("https://www.guardiansubscriptions.co.uk/Voucher?prom=faa03&pkgcode=ukx01&title=gv7&skip=1"),
+      sixday = Some("https://www.guardiansubscriptions.co.uk/Voucher?prom=faa03&pkgcode=ukx01&title=gv6&skip=1"),
+      weekend = Some("https://www.guardiansubscriptions.co.uk/Voucher?prom=faa03&pkgcode=ukx01&title=gv2&skip=1"),
+      sunday = Some("https://www.guardiansubscriptions.co.uk/Voucher?prom=faa03&pkgcode=ukx01&title=ov1&skip=1")
+    ))
+  }
+
+  val collectionPaper = Action { request =>
+    index(request, ShippingDetails(
+      name = "Paper voucher subscription",
```
Comment (marchibbins) [inline app/controllers/Shipping.scala]: "Paper Voucher Subscription"

File: app/controllers/Shipping.scala [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+        case "weekend" => shippingDetails.weekend
+        case "sunday" => shippingDetails.sunday
+      }
+      .flatten
+      .map(Redirect(_))
+      .getOrElse(Ok(views.html.shipping.index(shippingDetails)))
+
+  val collectionPaperDigital = Action { request =>
+    index(request, ShippingDetails(
+      name = "Paper + digital voucher subscription",
```
Comment (marchibbins) [inline app/controllers/Shipping.scala]: "Paper + Digital Voucher Subscription"

File: app/views/index.scala.html [hunk 9]
```diff
@@ -1,9 +1,126 @@
...
+                <div class="block__info">
+                    <ul class="block__list">
+                        <li class="block__list-item"><strong>Save up to 60%</strong>
+                            on the Guardian and Observer newspapers and digital daily edition</li>
+                        <li class="block__list-item">All supplements Monday to Sunday</li>
+                        <li class="block__list-item">Instant access to the daily edition and Guardian App Premium Tier</li>
+                    </ul>
+                </div>
+                <div class="block__footer">
+                    <a class="button button--large button--primary" href="https://www.guardiansubscriptions.co.uk/Voucher?prom=faa02&amp;pkgcode=ukx01&amp;title=gv7&amp;skip=1" data-test-id="subscriptions-uk-paper-digital">
```
Comment (marchibbins) [inline app/views/index.scala.html]: This should link to `/collection/paper-digital`

File: app/views/index.scala.html [hunk 10]
```diff
@@ -1,9 +1,126 @@
...
+                <div class="block__info">
+                    <ul class="block__list">
+                        <li class="block__list-item"><strong>Save up to 25%</strong>
+                            on the Guardian and Observer newspapers</li>
+                        <li class="block__list-item">Pick the package you want: Everyday, Sixday and Weekend</li>
+                        <li class="block__list-item">All supplements Monday to Sunday</li>
+                    </ul>
+                </div>
+                <div class="block__footer">
+                    <a class="button button--large button--primary" href="https://membership.theguardian.com/join?INTCMP=GU_SUBSCRIPTIONS_UK_PROMO" data-test-id="subscriptions-uk-paper">
```
Comment (marchibbins) [inline app/views/index.scala.html]: This should link to `/collection/paper`

File: app/views/index_intl.scala.html [hunk 11]
```diff
@@ -0,0 +1,79 @@
...
+                <ul class="block__list">
+                    <li class="block__list-item">Access to the daily UK edition, the digital version of the Guardian newspaper</li>
+                    <li class="block__list-item">Access to the Guardian App Premium Tier, an advert free experience</li>
+                    <li class="block__list-item">Available on Apple, Android and Kindle Fire</li>
+                    <li class="block__list-item"><strong>Save up to 75%</strong></li>
+                    <li class="block__list-item">14-day free trial</li>
+                </ul>
+            </div>
+            <div class="block__footer">
+                <a class="button button--large button--primary button--arrow" href="/@edition.id/digitalpack" data-test-id="subscriptions-@edition.id-digital">Subscribe now</a>
```
Comment (marchibbins) [inline app/views/index_intl.scala.html]: This should link to `/@edition.id/digital`

**Comment by marchibbins** [comment]
:+1: after comments

File: app/views/shipping/index.scala.html (hunk 4 above)
Comment (marchibbins) [inline app/views/shipping/index.scala.html]: `shipping.name` needs to be title case here.
